renovate.json

```
@@ -4,5 +4,15 @@
   ],
   "ignorePaths": [
     "go/*/Dockerfile"
+  ],
+  "enabledManagers": ["regex"],
+  "regexManagers": [
+    {
+      "fileMatch": ["^java/cloudbuild.yaml$", "^java/cloudbuild-test.yaml$"],
+      "matchStrings": ["substitutions:\n  _GRAALVM_VERSION: '(?<currentValue>.*?)'"],
```

Do we need `substitutions:`, as it requires that `_GRAALVM_VERSION` always be the first in the list of potentially multiple future values. Someone adding a new first entry would be unlikely to realize they've broken this regex check.
```suggestion
      "matchStrings": ["^  _GRAALVM_VERSION: '(?<currentValue>.*?)'"],
```

---

Good call! I didn't think of this scenario. Done with a minor modification that used ` _GRAALVM_VERSION` as opposed to `^  _GRAALVM_VERSION` as per testing in https://github.com/mpeddada1/renovate-bot-test/issues/1#issuecomment-1359716745